README.md

```diff
@@ -145,9 +145,14 @@ echo 8 > /sys/class/net/enp2s0f0/device/sriov_numvfs
 * `spoofchk` (string, optional): turn packet spoof checking on or off for the VF
 * `trust` (string, optional): turn trust setting on or off for the VF
 * `link_state` (string, optional): enforce link state for the VF. Allowed values: auto, enable, disable. Note that driver support may differ for this feature. For example, `i40e` is known to work but `igb` doesn't.
+* `min_tx_rate` (int, optional): change the allowed minimum transmit bandwidth, in Mbps, for the VF. Setting this to 0 disables rate limiting. Note: Only supported on Mellanox NICs.
 * `max_tx_rate` (int, optional): change the allowed maximum transmit bandwidth, in Mbps, for the VF. 
 Setting this to 0 disables rate limiting.
 
+Note: min_tx_rate is only supported on Mellanox NICs. Setting the min_tx_rate on Intel NICs to any value other than 0 
+will prevent the Pod from being created with "Invalid Argument".
+
+The recommendation is to not configure min_tx_rate in the net-attach-defs for Intel NICs.
```

Comment:
Lets remove this statement from here. This feature is supported by other NICs as well.
A more simpler statement like "support of this feature depends on NICs and drivers" would suffice.

Some more useful information would be good to add: the min_tx_rate should be <= max_tx_rate if user do want to set these.

Reply:
Done. Thanks for reviewing.